deepdrivemd/models/keras_cvae_stream/train.py

```diff
@@ -0,0 +1,151 @@
+import time
+import json
+from pathlib import Path
+from typing import Tuple, List, Optional
+import numpy as np
+from deepdrivemd.utils import Timer, timer, parse_args
+from deepdrivemd.data.api import DeepDriveMD_API
+from deepdrivemd.data.utils import get_virtual_h5_file
+from deepdrivemd.selection.latest.select_model import get_model_path
+from deepdrivemd.models.keras_cvae_stream.config import KerasCVAEModelConfig
+from deepdrivemd.models.keras_cvae_stream.model import conv_variational_autoencoder
+import subprocess
+import glob
+from aggregator_reader import *
+import os, sys
+
+def get_init_weights(cfg: KerasCVAEModelConfig) -> Optional[str]:
```

Comment:
This function is unused in the streaming implementation. It should be removed.

Reply:
Removed.